support-frontend/assets/helpers/user/user.js

```diff
@@ -58,6 +58,20 @@ const signOut = () => { window.location.href = getSignoutUrl(); };
 
 const doesUserAppearToBeSignedIn = () => !!cookie.get('GU_U');
 
+const getEmailValidatedFromUserCookie = (guuCookie: ?string) => {
+  if (guuCookie) {
+    const tokens = guuCookie.split('.');
+    try {
+      const parsed = JSON.parse(atob(tokens[0]));
+      return !!parsed[7]
```

Comment:
What are the chances of this index changing?

Reply:
It would break a lot of things, this stuff is hardcoded in identity

Reply:
got it

Reply:
I added inline documentation

Reply:
👍